.changeset/lemon-coats-camp.md

```diff
@@ -0,0 +1,11 @@
+---
+'@backstage/repo-tools': minor
+---
+
+Add new command options to the `api-report`
+
+- added `--allowWarnings` to continue processing packages if some packages have warnings
+- added `--omitMessages` to pass some warnings messages code to be omitted from the api-report.md files
```

Comment:
```suggestion
- added `--allow-warnings` to continue processing packages if some packages have warnings
- added `--omit-messages` to pass some warnings messages code to be omitted from the api-report.md files
```

Also implying that we should switch `omitMessages` to kebab, it's what we use everywhere else.